lib/widgets/header.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:geocoding/geocoding.dart';
+import 'package:get/get.dart';
+import 'package:intl/intl.dart';
+import 'package:weather_app/controller/global_controller.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class HeaderWidget extends StatefulWidget {
+  const HeaderWidget({super.key});
+
+  @override
+  State<HeaderWidget> createState() => _HeaderWidgetState();
+  Widget build(BuildContext context) {
+    return Container(
```

Comment:
Use SizedBox here instead
```suggestion
    return SizedBox(
```